CHANGELOG.md

```diff
@@ -1,4 +1,59 @@
+- Documentation of the new process system
+    [[#2751]](https://github.com/aiidateam/aiida_core/pull/2751)
+    [[#2752]](https://github.com/aiidateam/aiida_core/pull/2752)
+    [[#2753]](https://github.com/aiidateam/aiida_core/pull/2753)
+    [[#2754]](https://github.com/aiidateam/aiida_core/pull/2754)
+    [[#2802]](https://github.com/aiidateam/aiida_core/pull/2802)
+    [[#2805]](https://github.com/aiidateam/aiida_core/pull/2805)
+    [[#2806]](https://github.com/aiidateam/aiida_core/pull/2806)
+    [[#2848]](https://github.com/aiidateam/aiida_core/pull/2848)
+    [[#2852]](https://github.com/aiidateam/aiida_core/pull/2852)
+
+### Bugs
+- Fix bug in process control in direct scheduler [[#2721]](https://github.com/aiidateam/aiida_core/pull/2721)
+- Do not treat `None` like values as `None` in `Config.option_set` [[#2731]](https://github.com/aiidateam/aiida_core/pull/2731)
+- `Orbital` code is refactored and some bugs have been fixed [[#2737]](https://github.com/aiidateam/aiida_core/pull/2737)
+- Catch `IOError` in `CalcJobNode` methods to get scheduler output [[#2750]](https://github.com/aiidateam/aiida_core/pull/2750)
+- Prevent additional incoming links for a stored `ProcessNode` [[#2777]](https://github.com/aiidateam/aiida_core/pull/2777)
+- Fix bug in `KpointsData.reciprocal_cell()` [[#2779]](https://github.com/aiidateam/aiida_core/pull/2779)
+- Raise `InvalidOperation` for global `submit` in nested process [[#2781]](https://github.com/aiidateam/aiida_core/pull/2781)
+- Bug fixes and cleanup for `aiida.manage.external.Postgres` [[#2769]](https://github.com/aiidateam/aiida_core/pull/2769)
+- Fix bug in `QuerBuilder.first()` for multiple projections [[#2824]](https://github.com/aiidateam/aiida_core/pull/2824)
+- Require `store_provenance=True` in `submit` or raise [[#2828]](https://github.com/aiidateam/aiida_core/pull/2828)
+- Fix bug in Django migration `0023_calc_job_option_attribute_keys` [[#2798]](https://github.com/aiidateam/aiida_core/pull/2798)
+- Fix bug in SqlAlchemy migration `7ca08c391c49_calc_job_option_attribute_keys` [[#2798]](https://github.com/aiidateam/aiida_core/pull/2798)
+- REST API: Adapt calculation input/output file endpoints to new repository interface [[#2840]](https://github.com/aiidateam/aiida_core/pull/2840)
+- REST API: fix and update the `verdi restapi` command[[#XXXXX]](https://github.com/aiidateam/aiida_core/pull/XXXXX)
+- Fix bug in creation of `SinglefileData` in `retrieve_calculation` [[#2860]](https://github.com/aiidateam/aiida_core/pull/2860)
+- Fix calling show_mpl() on a BandsData object [[#2818]](https://github.com/aiidateam/aiida_core/pull/2818)
+- Use binary mode for local copy list in `upload_calculation` [[#2748]](https://github.com/aiidateam/aiida_core/pull/2748)
+- Respect option `-p/--profile` in `verdi profile show` [[#2876]](https://github.com/aiidateam/aiida_core/pull/2876)
+
+### Backwards incompatible
```

Comment:
this misses changing the `profile` from an argument to an option in `verdi setup`